src/Microsoft.Framework.ConfigurationModel.Json/project.json

```diff
@@ -9,6 +9,10 @@
     "frameworks": {
         "net45": { },
         "dnx451": { },
-        "dnxcore50": { }
+        "dnxcore50": {
+            "dependencies": {
+                "System.Dynamic.Runtime": "4.0.10-*"

```

Comment:
Or, I should clarify: when I use a particular type from json.net, the compiler complains about that missing reference